shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -144,6 +148,10 @@ class GlutenConfig(conf: SQLConf) extends Logging {
   def enableLogicalJoinOptimize: Boolean =
     conf.getConfString("spark.gluten.sql.columnar.logicalJoinOptimizeEnable", "false").toBoolean
 
+  // When enabled, only scan and the filter after scan will be offloaded to native.
+  def enableScanOnly: Boolean =
```

Comment:
Can you add the config to readme as well? highlight the config will overwrite all other operators' enabling. Just offload parquet scan and filter pushdown.

Reply:
Updated doc.